VTEX - Marketplace APIs.json

```diff
@@ -3379,44 +3039,509 @@
+                                                    "listPrice": 0.0,
+                                                    "priceWithoutDiscount": 0.0,
+                                                    "availableQuantity": 0
+                                                }
+                                            }
+                                        ]
+                                    },
+                                    {
+                                        "productId": "1561208",
+                                        "skuId": "1559163",
+                                        "nameComplete": "Blusa Cardigan Manga Longa Vestem - Feminina - CINZA PP",
+                                        "mainImage": {
+                                            "imagePath": "~/arquivos/ids/8567753-#width#-#height#/image-05b40d3167b5b7ed42285124f6862f12.jpg"
+                                        },
+                                        "sellersOffers": [
+                                            {
+                                                "sellerId": "1",
+                                                "sellerSkuId": "1559161",
+                                                "salesChannelOffer": {
+                                                    "salesChannelId": "1",
+                                                    "salesChannelName": "Principal",
+                                                    "price": 0.0,
+                                                    "listPrice": 0.0,
+                                                    "priceWithoutDiscount": 0.0,
+                                                    "availableQuantity": 0
+                                                }
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        }
+                    }
+                }
             }
-        }
-    },
-    "tags": [
-        {
-            "name": "Sellers",
-            "description": "Get sellers' data"
-        },
-        {
-            "name": "Seller Invite",
-            "description": "Used to invite sellers and configure their accounts"
-        },
-        {
-            "name": "Seller Commissions",
-            "description": "Get sellers' data"
-        },
-        {
-            "name": "Notification"
         },
-        {
-            "name": "Matched Offers",
-            "description": ""
-        },
-        {
-            "name": "Sales Channel Mapping",
-            "description": "Mapping between a marketplace's sales channel and a seller's affiliate."
+            "securitySchemes": {
+                "appKey": {
+                    "type": "apiKey",
+                    "in": "header",
+                    "name": "X-VTEX-API-AppKey"
```

Comment:
Adicionar
"description": "Unique identifier of the [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys)."